cmd/kubeadm/app/features/features.go

```diff
@@ -35,15 +35,19 @@ const (
 
 	// StoreCertsInSecrets is alpha in v1.8
 	StoreCertsInSecrets = "StoreCertsInSecrets"
+
+	//SupportIPVSProxyMode is alpha in v1.8
```

Comment:
nit: `// IPVSProxyMode`
please change the name to `IPVSProxyMode` and add one space to the comment.

Reply:
also, should we make this beta in v1.9?

Reply:
Yes, this gone be beta in v1.9.